jerry-core/ecma/builtin-objects/ecma-builtin-object.c

```diff
@@ -291,7 +291,12 @@ ecma_builtin_object_set_integrity_level (ecma_object_t *obj_p, /**< object */
 #endif /* ENABLED (JERRY_ES2015_BUILTIN_PROXY) */
 
   /* 6. */
+#if ENABLED (JERRY_ES2015)
+  ecma_collection_t *props_p = ecma_op_object_get_property_names (obj_p, ECMA_LIST_CONVERT_FAST_ARRAYS
```

Comment:
I'd prefer guarding only the options bits rather than the whole statement. Other looks good to me.

Reply:
Thanks for the review, I put _ECMA_LIST_SYMBOLS_ option to if statement as you suggested. I tried also with putting both options, but it didn't look good. What are the guidelines for situation like this? I couldn't find anything similar in existing code.

Reply:
How about this?

```c
uint32_t opts = convert_fast_array
#if es2015
 opts |= list_symbols
#endif
```
Now you can pass `opts` as a second argument to the property name listing function.

Reply:
I thought about this, but can we be sure, that compilator would optimize it? It's another variable and I wanted to avoid using it. But if it's okay, then I agree that it would be the best solution :)

Reply:
It's only a temporary variable with a constant immediate value ( in both cases) so it cannot cause any problem :)

Reply:
I checked binary executable files, and there is no change in size on release build. Only 16 additional bytes on debug build. So you are right - it's fine ;)